CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Display and Debug for SysControlAddr now includes all fields.
   ([#837](https://github.com/nix-rust/nix/pull/837))
+- `nix::net::if_::InterfaceFlags` now support `IFF_NO_PI/IFF_TUN/IFF_TAP` flags on linux-like system.
```

Comment:
This is technically an addition since the interface didn't change, but new functionally was added. Please move it to the Added section.

Also change "system" to "systems"